cmd/build/helmify/static/templates/namespace-post-install.yaml

```diff
@@ -114,3 +117,4 @@ subjects:
     namespace: {{ .Release.Namespace | quote }}
 {{- end }}
 {{- end }}
+
```

Comment:
nit: can you remove this extra new line?

Reply:
removed